4_Methodology/BS_SD.adoc

```diff
@@ -1314,17 +1314,14 @@ However, it is important to be able to compare the resistance of various systems
 
 The levels are as follows:
 
---
 [loweralpha]
-. _Immediate_ is for 2D face, signature image, and voice. Samples of these modalities can be collected without difficulty, even without direct contact with an enrolled data subject (an exploration of the web and the social networks and so forth).
-. _Easy_ is for fingerprint. Latent fingerprints are often left on objects the enrolled data subject had in hand, but need to be revealed, acquired and the corresponding images need a preprocessing.
-. _Moderate_ is for 3D face, dynamic signature, and 3D fingerprint. 3D images require multiple acquisitions, probably in a controlled way, without the collaboration of an enrolled data subject but probably with a direct contact with them.
-. _Difficult_ is for iris and vein. Iris images can be acquired with a high resolution camera, but with some difficulties to get a complete high quality image without the cooperation of an enrolled data subject. Veins are a hidden characteristic, but infra-red cameras, close to them, can acquire images to be used.      
---
+. _Without notice_ is for samples for making an artefact that can be collected without any contact with an enrolled subject. For example, 2D face images uploaded on the Internet and latent fingerprint images on a glass can be collected without notice of the subject.
+. _Non-cooperative_ is for samples for making an artefact that need to be collected directly from an enrolled subject in a short period of time without full cooperation from the subject. For example, iris or vein images need to be acquired with a high resolution or infra-red camera, however, such images can be taken in a moment without full control of the subject.
```

Comment:
```suggestion
. _Non-cooperative_ is for samples for making an artefact that need to be collected directly from an enrolled subject in a short period of time without full cooperation from the subject. For example, iris or vein images need to be acquired with a high resolution or infrared camera, however, such images can be taken in a moment without full control of the subject.
```

Reply:
infrared seems to be the common spelling today, not infra-red. Even dictionary.com points infra-red to the infrared description and doesn't even try to show the work with the hyphen (outside some search result examples).